Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM ubuntu:20.04
+
+ENV TERM="xterm-color"
+ARG USER_ID=999
+ARG GROUP_ID=999
+ARG USER=docker
+ARG PWD=docker
+ARG HOME="/home/$USER"
+
+# Create our group and user so we can login as ourself. Useful for ssh.
+RUN groupadd -g $GROUP_ID $USER && \
+    useradd -g $GROUP_ID -m -s /bin/bash -u $USER_ID $USER
+
+# Suppresses prompt for timezone
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Yocto dependencies
+RUN apt-get update
+RUN apt-get -y upgrade
+RUN apt-get -y install gawk make wget tar bzip2 gzip python3 unzip perl patch \
+    diffutils diffstat git cpp gcc texinfo chrpath \
+    ccache socat \
+    python3-pexpect findutils file cpio python python3-pip \
+    python3-jinja2 xterm bash
+
+# Unable to locate packages. These were likely packaged with other packages
+# gcc-c++  glibc-devel perl-Data-Dumper perl-Text-ParseWords perl-Thread-Queue perl-bignum
+# which xz SDL-devel python3-GitPython
+
```

Comment:
Not all packages are provided by apt / snap repositories (ie Ubuntu/Debian doesn't directly support them).  Some, such as GitPython are available via Python's pip packaging.  Other packages, particularly dev packages, get broken down into small packages.  Often, there is a meta-package but I couldn't find an obvious one for libsdl-\*. 

Anyway, for the SDL packages, there is libsdl-*, libsdl1\*-\* and libsdl2-\*.   The package 'python-sdl2' might get what you need but I am not familar with libsdl\*.   Apt tools such as 'apt list | grep -i some string' and 'apt-file list <package>' will get package names and then the contents of a package. 

 'apt-file' is not installed by default and needs to have 'apt-list update' run after it is first installed.

You may or may not recall but installing pip packages should be done by the target $USER unless there is a need to install them for the whole system.

Reply:
These were packages that were part of the yocto getting started guild and part of the `apt-get install` command. So its interesting they put packages that aren't part of `apt`. Either way it builds so I'm happy.